### ISSUE TITLE
Loss becomes NaN

### ISSUE THREAD

**Issue by LordLiang** [hoidini/HOIDiNi#4]
Hi, thank you for this excellent work! 
When I run the training code `./scripts/train_cphoi.sh`, the loss suddenly becomes NaN after the 625th epoch. What could be the possible reasons for this?
<img width="1857" height="749" alt="Image" src="https://github.com/user-attachments/assets/a041fa7f-084c-4127-931b-7513d0531bab" />

**Comment by Pcc123456** [comment]
I also replicated the code, but I didn't encounter this problem.

**Comment by LordLiang** [comment]
> I also replicated the code, but I didn't encounter this problem.

HI. Can you kindly tell me about your environment? (GPU, CUDA, Pytorch)

**Comment by LordLiang** [comment]
I've solved this problem. It was an issue with the PyTorch version.
Because I'm using a damn 5090 GPU, I have to use CUDA 12.8.
Previously, I had PyTorch 2.7.1 installed, but the loss was NaN.
Now that I've installed PyTorch 2.8.0, it works perfectly!
Please keep far away from PyTorch 2.7.1!
^.^
